openedx_webhooks/decorators.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
I didn't find anything readily. The only thing I know of is `django.utils.functional.lazy`, but maybe you have suggestions?